mappings/net/minecraft/entity/passive/CatEntity.mapping

```diff
@@ -24,12 +27,14 @@ CLASS net/minecraft/unmapped/C_rgermtxr net/minecraft/entity/passive/CatEntity
 	FIELD f_lzbtguaw prevTailCurlAnimation F
```

Comment:
previous instead of prev? You can keep prev though just thought it was a bit odd but this is fine